core/src/main/java/org/hibernate/ogm/massindex/impl/OptionallyWrapInJTATransaction.java

```diff
@@ -36,7 +37,7 @@
  */
 public class OptionallyWrapInJTATransaction implements ModelConsumer {
 
-	private static final Log log = LoggerFactory.make();
+	private static final Log log = LoggerFactory.make( Log.class, MethodHandles.lookup() );
```

Comment:
Not related to this patch but the fact that the initialization of the logger was different made me take a look to what we are doing.

Thinking out loud, I'm wondering if we could get rid of these copies of Search classes. I think I'll take a look as part of the ORM 5.2/Search 5.8 upgrade.

Reply:
We can get rid of these but we need to create similar error log in our logger, not sure why we used the one from search, probably a mistake in the import. It's OK if you want to have a lok

Reply:
I'm not talking about getting rid of the logger class. I'm talking about getting rid of the whole `OptionallyWrapInJTATransaction` class (and probably some others in the package). AFAICS, the Search API are far more customizable that they used to be.

Not sure it's possible but it's worth having a fresh look at it, I think.